modulefiles/module_base.jet

```diff
@@ -1,32 +1,24 @@
+#%Module######################################################################
 ##
-##      nems prerequisites
+##      FV3GFS prerequisites
 ##
 
-proc ModulesHelp {} {
-     puts stderr "The prerequisites for compiling or running FV3 on Jet. "
-}
+# Load hpc-stack
+module use /lfs4/HFIP/hfv3gfs/nwprod/hpc-stack/libs/modulefiles/stack
+module load hpc/1.1.0
 
-module load newdefaults intel/15.0.3.187 impi/5.1.3.181 szip hdf5 netcdf4/4.2.1.1
-
-module use /lfs3/projects/hfv3gfs/nwprod/lib/modulefiles
-module load bacio/v2.0.2
-module load sp/v2.0.2
-module load ip/v2.0.0
-module load w3nco/v2.0.6
-module load w3emc/v2.2.0
-module load nemsio/v2.2.2
-
-#set NCEPLIBS $::env(NCEPLIBS)
-#module use $NCEPLIBS/modulefiles
-#module load esmf/7.1.0r_impi
-
-module use /lfs3/projects/hwrf-vd/soft/modulefiles
-module load prod_util
-module load grib_util
+# Load intel compiler and mpi
+module load hpc-intel/18.0.5.274
+module load hpc-impi/2018.4.274
+module load wgrib2/2.0.8
 module load hpss
-
-# mpiserial on Jet includes a cfp wrapper
-module load mpiserial
-
+module load nco/4.9.1
+module load cdo/1.9.5
 module load gempak/7.4.2
-
+module load prod_util/1.2.2
+module load grib_util/v1.1.1
```

Comment:
Some of these module versions are still not the correct ones located in hpc-stack. Please compare this modulefile with the one from Hera that already uses hpc-stack and ensure the module versions are the same for all that available under hpc-stack.

Reply:
I compared each one of the module. Found the following module on Hera is not in hpc-stack:
hpss/hpss
nco/4.9.1
cdo/1.9.5
gempak/7.4.2
intel/18.0.5.274
impi/2018.4.274

I made update to include:
grib_util/1.2.2
g2tmpl/1.10.0

Reply:
The g2tmpl version should be the same as the other machines (1.9.1).

Reply:
> The g2tmpl version should be the same as the other machines (1.9.1).

Hera used g2tmpl/1.10.0 because module upp/10.0.6:

module load upp/10.0.6
The following have been reloaded with a version change:
  1) g2/3.4.1 => g2/3.4.3     2) g2tmpl/1.9.1 => g2tmpl/1.10.0     3) w3emc/2.7.3 => w3emc/2.9.0


Therefore, I used g2tmpl/1.10.0 on JET

Reply:
Okay, we'll have to update the other modulefiles separately then.